actionpack-xml_parser.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = %w( README.md )
   s.rdoc_options.concat ['--main',  'README.md']
 
-  s.add_dependency('actionpack', '>= 4.0.0', '< 5')
+  s.add_dependency('actionpack', '~> 5.x')
```

Comment:
You commented on a wrong line.

But sure. @eileencodes do you mind changing the version in another commit as well?